commands/login_sso.go

```diff
@@ -0,0 +1,105 @@
+package commands
+
+import (
+	"fmt"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/fatih/color"
+	"github.com/giantswarm/gsctl/client"
+	"github.com/giantswarm/gsctl/config"
+	"github.com/giantswarm/gsctl/pkce"
+	"github.com/spf13/cobra"
+)
+
+const (
+	activityName = "sso"
```

Comment:
Seems like this is unused.

IMO we can use the same activity name as for the standard login, so simply get rid of this constant.